### PR TITLE
Fixes typo on setup-node's version

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Check latest version:
 ```yaml
 steps:
 - uses: actions/checkout@v2
-- uses: actions/setup-node@v2
+- uses: actions/setup-node@v1
```
Comment (bradennapier) [inline README.md:50]:
```suggestion
- uses: actions/setup-node@v2.0.0
```

Comment (josejulio) [reply README.md:50]: Ah, `v2` gave me an error and I saw that it was still beta.

Comment (bradennapier) [reply README.md:50]: You should merge the suggestion so they can accept it (hopefully) :)

**Comment by josejulio** [comment]
Using `v2.0.0` instead as suggested by @bradennapier

**Comment by bryanmacfarlane** [comment]
`v2` was intentional and is consistent with the starter templates

**Pull request closed without merging**